lib/core/models/quick_link_model.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:flutter/material.dart';
```

Comment:
Rename the file quick_link should be enough ;)

Reply:
done